services/ui-src/src/views/Admin/Admin.tsx

```diff
@@ -78,6 +94,7 @@ const sx = {
   root: {
     flexShrink: "0",
   },
+  errorAlert: {},
```

Comment:
Left this here 'cause I figure we're gonna use it once design gets back to me. 🤷